app/scripts/lib/metasea.js

```diff
@@ -0,0 +1,64 @@
+module.exports = seajack;
+var GUN = require('gun/gun.min.js');
```

Comment:
Imported but not used. Is this MVP only about using SEA?

If so, does this mean there is not currently a gun data container? Or is that held in page? (in page would be less efficient, because it would mean sites are not sharing a GUN instance, unless you're telling me inter-tab GUN is so efficient that it doesn't matter. I still would probably rather that we weren't loading a whole extra lib into every page, but depending on its size/efficiency maybe it could be worthwhile).

Reply:
This will go away as teams commit to SEA. Then my community can yank it out (they already want to) and have it standalone. As for right now, I reuse a couple JS utilities that GUN uses to save a few bytes.

Data :( provided by network or localStorage/indexedDB in page - stupid CORS. MM could certainly let us get around that (yay!) but user would already need GUN plugin running - but lets keep that as a separate issue from SEA.